<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_issue</class>
 <widget class="QMainWindow" name="book_issue">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2410</width>
    <height>1047</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Issue</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bookholics.qrc">
    <normaloff>:/image/1.png</normaloff>:/image/1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>521</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/image/logo.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>511</width>
      <height>351</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>492</width>
       <height>477</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_book_id">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string>Enter Book ID</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Book ID*</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Name of Book</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_book_author">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Edition</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_book_name">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Name of Author</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_edition">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Sylfaen</family>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Issued To</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QComboBox" name="user_type">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Borrow ID</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="lineEdit_borrow_id">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
          <property name="placeholderText">
           <string> Enter Unique ID</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Full Name</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="lineEdit_full_name">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Department</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLineEdit" name="lineEdit_class">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QLineEdit" name="lineEdit_dept">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_class">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Class</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLineEdit" name="lineEdit_division">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_div">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Division</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_12">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Issue As</string>
          </property>
         </widget>
        </item>
        <item row="10" column="1">
         <widget class="QComboBox" name="comboBox_issue_as">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:2px;</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Issued Date</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_3"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_issue_date">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="13" column="0">
         <layout class="QFormLayout" name="formLayout_6"/>
        </item>
        <item row="11" column="1">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Due Date</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_5"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_due_date">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding:2px;
border-style:solid;
border:2px solid blue;
border-radius:8px;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="12" column="0">
         <layout class="QFormLayout" name="formLayout_7"/>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">padding:4px;</string>
        </property>
        <property name="text">
         <string>Issue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="bookholics.qrc"/>
 </resources>
 <connections/>
</ui>
